### PR TITLE
fix: typos inside of public markdown docs

### PR DESCRIPTION
**Pull request by JessicaSachs**
I didn't realize we'd get so many typo PRs after launching the website 😅. I don't think any of these are duplicates.

The changes are scoped to the public markdown documents, not code, so there shouldn't be much risk.

### CODE REVIEW

File: src/api/application.md [hunk 1]
```diff
@@ -438,11 +438,11 @@ Adjusts template whitespace handling behavior.
 
 - **Details**
 
-  Vue removes / condenses whitespaces in templates to produce more efficient compiled output. The default strategy is "condense", with the following behavior:
+  Vue removes / condenses whitespace in templates to produce more efficient compiled output. The default strategy is "condense", with the following behavior:
 
-  1. Leading / ending whitespaces inside an element are condensed into a single space.
-  2. Whitespaces between elements that contain newlines are removed.
-  3. Consecutive whitespaces in text nodes are condensed into a single space.
+  1. Leading / ending whitespace inside an element is condensed into a single space.
```
Comment (NataliaTepluhina) [inline src/api/application.md:443]: I believe this case is correct as we're speaking about plural form here

File: src/api/component-instance.md [hunk 2]
```diff
@@ -251,10 +251,10 @@ Imperative API for creating watchers.
   Stopping the watcher:
 
   ```js
-  const unwatch = this.$watch('a', cb)
+  const stopWatching = this.$watch('a', cb)
```
Comment (NataliaTepluhina) [inline src/api/component-instance.md:254]: **question**: Could you please provide more context on why this change is necessary?

**Review by NataliaTepluhina — COMMENTED** [review]
@JessicaSachs thank you for catching so many typos! I've left comments on the cases I disagree with, rest of the fixes look great to me 👍🏻

File: src/about/coc.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
...
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socioeconomic status, nationality, personal appearance, race, religion, political party, or sexual identity and orientation. Note, however, that religion, political party, or other ideological affiliation provide no exemptions for the behavior we outline as unacceptable in this Code of Conduct.
```
[the remark this answers is not in the document]
Comment (JessicaSachs) [reply src/about/coc.md:5]: Oh, idc then. Lgtm 👍

File: src/api/application.md (hunk 1 above)
Comment (JessicaSachs) [reply src/api/application.md:443]: Whitespace is usually an uncountable noun https://english.stackexchange.com/a/25371 thus the singular usage.

File: src/api/component-instance.md (hunk 2 above)
Comment (JessicaSachs) [reply src/api/component-instance.md:254]: Unwatch didn't pass spellcheck. I didn't realize it was even a word. https://en.m.wiktionary.org/wiki/unwatch it's marked as rare because it isn't used often.

`stopWatching` is much clearer and passes spellcheck.

File: src/about/coc.md (hunk 3 above)
Comment (JessicaSachs) [inline src/about/coc.md]:
```suggestion
In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, political party, or sexual identity and orientation. Note, however, that religion, political party, or other ideological affiliation provide no exemptions for the behavior we outline as unacceptable in this Code of Conduct.
```

File: src/api/application.md (hunk 1 above)
Comment (JessicaSachs) [reply src/api/application.md]: Swapped it for `whitespace characters` so we can keep it plural.

File: src/api/component-instance.md (hunk 2 above)
Comment (JessicaSachs) [reply src/api/component-instance.md:254]: I don't feel strongly about it.
